config/forms/2021_2022.json

```
@@ -5396,40 +5396,79 @@
                 }
               }
             },
-            "access_needs": {
+            "access_needs_exist": {
               "header": "",
               "description": "",
               "questions": {
-                "accessibility_requirements": {
+                "housingneeds": {
                   "header": "Does anybody in the household have any disabled access needs?",
                   "hint_text": "",
-                  "type": "checkbox",
-                  "check_answer_label": "Anybody in household with disabled access needs",
+                  "type": "radio",
+                  "check_answer_label": "Anybody with disabled access needs",
+                  "answer_options": {
+                    "1": {
+                      "value": "Yes"
+                    },
+                    "2": {
+                      "value": "No"
+                    },
+                    "divider": {
+                      "value": true
+                    },
+                    "3": {
+                      "value": "Don’t know"
+                    }
+                  }
+                }
+              }
+            },
+            "type_of_access_needs": {
+              "header": "Disabled access needs",
+              "description": "",
+              "questions": {
+                "housingneeds_type": {
+                  "header": "What type of access need do they have?",
```

```suggestion
                  "header": "What types of access need do they have?",
```

I think this should be plural.